### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mw-hrastega**
adjustments for v1 actions

### CODE REVIEW

**Review by acampbel — APPROVED** [review]

**Review by acampbel — COMMENTED** [review]
While we are changing this repo should we update the linux VMs to be Ubuntu 18.04, 20.04 or something like ubuntu latest? IN at least gtihub actions we jsut got a notice that 16.04 will no longer be used

**Comment by mw-hrastega** [comment]
> While we are changing this repo should we update the linux VMs to be Ubuntu 18.04, 20.04 or something like ubuntu latest? IN at least gtihub actions we jsut got a notice that 16.04 will no longer be used

Good point, Andy. I updated the versions and labels.

File: README.md [hunk 1]
```diff
@@ -39,12 +39,12 @@ steps:
 ```yml
 version: 2.1
 orbs:
-  matlab: mathworks/matlab@0.1.4
-  codecov: codecov/codecov@1.1.1
+  matlab: mathworks/matlab@0.4.0
+  codecov: codecov/codecov@1.2.0
```
Comment (mcafaro) [inline README.md]:
```suggestion
  matlab: mathworks/matlab@0
  codecov: codecov/codecov@1
```

I would just specify the major version on these so this document doesn't get out-of-date too often.

**Review by mcafaro — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mw-hrastega) [reply README.md]: Good idea.

**Pull request merged**